app/addons/replication/controller.js

```diff
@@ -257,8 +254,6 @@ export default class ReplicationController extends React.Component {
     } else {
```

Comment:
Very minor but could you remove the `else` and just `return [{'name': 'Replication'}];` by default